test/v2/mocks/services/MockCreateForeignPropertyPeriodSummaryService.scala

```diff
@@ -35,17 +32,15 @@ trait MockCreateForeignPropertyPeriodSummaryService extends MockFactory {
   object MockCreateForeignPropertyService {
 
     def createForeignProperty(requestData: CreateForeignPropertyPeriodSummaryRequest)
-      : CallHandler[Future[Either[ErrorWrapper, ResponseWrapper[CreateForeignPropertyPeriodSummaryResponse]]]] = {
+      : CallHandler[Future[CreateForeignPropertyPeriodSummaryServiceOutcome]] = {
```

Comment:
can we run the formatter on these files? looks strange to have the colon as the first character on a new line